org.sugarlabs.falabracmanActivity.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "org.sugarlabs.falabracmanActivity",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
```

Comment:
Can you try it with 3.36 ? It's the latest.

Reply:
i tried with 3.36 but it is showing following error 
![Screenshot from 2020-03-20 19-33-25](https://user-images.githubusercontent.com/40318927/77173354-643d1080-6ae5-11ea-9423-72c5d9815bf6.png)
however it is working fine with 3.34